components/phpunit_bridge.rst

```diff
@@ -73,15 +73,18 @@ in the **Unsilenced** section of the deprecation report.
 Mark Tests as Legacy
 --------------------
 
-There are four ways to mark a test as legacy:
+There are three ways to mark a test as legacy:
 
 * (**Recommended**) Add the ``@group legacy`` annotation to its class or method;
 
 * Make its class name start with the ``Legacy`` prefix;
 
-* Make its method name start with ``testLegacy*()`` instead of ``test*()``;
+* Make its method name start with ``testLegacy*()`` instead of ``test*()``.
 
-* Make its data provider start with ``provideLegacy*()`` or ``getLegacy*()``.
+.. note::
+
+    You can make your data provider start with ``provideLegacy*()`` or ``getLegacy*()``
```

Comment:
I'd like to have more information about this:

* If I name a data provider as `provideLegacy()` and `getLegacy()` ... will the tests that use that provider be considered legacy automatically?
* If I declare a test as legacy ... am I required to call its data provider as `provideLegacy()` and `getLegacy()` too?